templates/partials/menu.html

```diff
@@ -1,12 +1,13 @@
-<div class="d-flex justify-content-center">
-    <nav class="navbar navbar-expand-lg  menu">
+<div class="d-flex justify-content-start justify-content-sm-start justify-content-md-center">
```

Comment:
In the `md` size it should either be justified to the start as well, or it should not shift left when opened.
I suggest justifying to the start.